internal/model/service.go

```diff
@@ -67,7 +67,7 @@ type Step struct {
 
 	// Optional. If not specified, this step runs on every change.
 	// If specified, we only run the Cmd if the trigger matches the changed file.
-	Trigger PathMatcher
+	Trigger []PathMatcher
```

Comment:
per discussion on previous PR

Reply:
wait, what? you can have a PathMatcher that matches multiple files, so why would you need a slice of PathMatchers here?

Reply:
yeahhh good point. Okay I'll change it back.